app/scripts/models/auth_brokers/fx-desktop-v2.js

```diff
@@ -15,46 +15,28 @@ define(function (require, exports, module) {
 
   const _ = require('underscore');
   const FxSyncWebChannelAuthenticationBroker = require('./fx-sync-web-channel');
-  const HaltBehavior = require('views/behaviors/halt');
-  const NullBehavior = require('views/behaviors/null');
-  const p = require('lib/promise');
+  const HaltIfBrowserTransitions = require('views/behaviors/halt-if-browser-transitions');
 
-  var proto = FxSyncWebChannelAuthenticationBroker.prototype;
+  const proto = FxSyncWebChannelAuthenticationBroker.prototype;
+  const defaultBehaviors = proto.defaultBehaviors;
 
-  var FxDesktopV2AuthenticationBroker = FxSyncWebChannelAuthenticationBroker.extend({
-    defaultBehaviors: _.extend({}, proto.defaultBehaviors, {
-      // about:accounts displays its own screen after sign in, no need
-      // to show anything.
-      afterForceAuth: new HaltBehavior(),
-      // about:accounts displays its own screen after password reset, no
-      // need to show anything.
-      afterResetPasswordConfirmationPoll: new HaltBehavior(),
-      // about:accounts displays its own screen after sign in, no need
-      // to show anything.
-      afterSignIn: new HaltBehavior(),
-      // the browser is already polling, no need for the content server
-      // code to poll as well, otherwise two sets of polls are going on
-      // for the same user.
-      beforeSignUpConfirmationPoll: new HaltBehavior()
+  const FxDesktopV2AuthenticationBroker = FxSyncWebChannelAuthenticationBroker.extend({
+    defaultBehaviors: _.extend({}, defaultBehaviors, {
+      afterForceAuth: new HaltIfBrowserTransitions(defaultBehaviors.afterForceAuth),
+      afterResetPasswordConfirmationPoll: new HaltIfBrowserTransitions(defaultBehaviors.afterResetPasswordConfirmationPoll),
+      afterSignIn: new HaltIfBrowserTransitions(defaultBehaviors.afterSignIn),
+      beforeSignUpConfirmationPoll: new HaltIfBrowserTransitions(defaultBehaviors.beforeSignUpConfirmationPoll),
     }),
 
     defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
+      browserTransitionsAfterEmailVerification: true,
       chooseWhatToSyncCheckbox: false,
       chooseWhatToSyncWebV1: true,
       openWebmailButtonVisible: true
     }),
 
     type: 'fx-desktop-v2',
 
-    afterResetPasswordConfirmationPoll (/*account*/) {
```

Comment:
See [the comment below](https://github.com/mozilla/fxa-content-server/pull/5472/files#r138324319) as to why this function was removed.